CONTRIBUTING.md

```diff
@@ -43,4 +43,4 @@ following contribution guidelines:
 2. Create a new folder under `$GEOCAT_VIZ/src/geocat/viz/` and implement your plotting function inside, making
 use of the pre-defined parent classes if appropriate
 
-3. Add the function to the namespace by adding it to `$GEOCAT_VIZ/src/geocat/viz/__init__.py`
+3. Add the function to the namespace by adding it to `$GEOCAT_VIZ/docs/user_api/index.rst`
```

Comment:
This change isn't technically correct. Adding functions to the namespace and adding them to the documentation are two separate parts. 

The way this package is set up, there isn't currently a need to individually add each new function to the `__init__.py` file as described because we're importing everything under each `.py` file, but yeah, adding to the namespace technically doesn't have anything to do with adding it to the documentation (apart from being able to import it in the documentation)

Reply:
Ahh. So I was confused by that step since nothing was added there. I will fix this wording. And the functions all seem to use a different docstring formatting in viz.